src/services/ghost/classic-auto-complete/GhostInlineCompletionProvider.ts

```diff
@@ -123,8 +121,8 @@ export class GhostInlineCompletionProvider implements vscode.InlineCompletionIte
 	public suggestionsHistory: FillInAtCursorSuggestion[] = []
 	/** Tracks all pending/in-flight requests */
 	private pendingRequests: PendingRequest[] = []
-	private holeFiller: HoleFiller
-	private fimPromptBuilder: FimPromptBuilder
+	public holeFiller: HoleFiller
+	public fimPromptBuilder: FimPromptBuilder
```

Comment:
maybe add a comment here why we made this public

```suggestion
	public holeFiller: HoleFiller // publicly exposed for Jetbrains autocomplete code
	public fimPromptBuilder: FimPromptBuilder // publicly exposed for Jetbrains autocomplete code
```